autoinstall_templates/sample.seed

```diff
@@ -117,6 +117,10 @@ d-i pkgsel/include string ntp ssh wget
 # Note: options passed to the installer will be added automatically.
 d-i debian-installer/add-kernel-opts string $kernel_options_post
 
+# Grub settings
+d-i grub-installer/grub2_instead_of_grub_legacy boolean true
+d-i grub-installer/bootdev string /dev/[sv]da
```

Comment:
Right now I have no question on ubuntu server 16.10 amd64 using a kvm guest [1], so I still fail to see which debian/ubuntu releases are affected as asked in the first comment.
Right now, this patch will fails on xen guest because it uses /dev/xvda, but also on dmraid that might provides various style of devices names. Better to let the installer automatically fetch the best bootloader/device on the sample.seed at least.

[1] the guest failed to boot after activating the lvm, but it means bootloader was installed as appropriate at least (so it's a different issue).